### ISSUE TITLE
Replace all uses of BETWEEN

### ISSUE THREAD

**Issue by schuemie** [OHDSI/QueryLibrary#4]
Several queries use the SQL operator `BETWEEN`, for example [this query](https://github.com/OHDSI/QueryLibrary/blob/master/inst/shinyApps/QueryLibrary/queries/general/G01.md). Even though this translates without error to the different SQL dialects, it means different things. In some dialects the start and end dates are included. In some dialects the end date isn't included, etc. It is better to be explicit, so instead of 
```sql
WHERE my_date BETWEEN start_date AND end_date
```
we should use 
```sql
WHERE my_date >= start_date AND my_date <= end_date
```

**Comment by PRijnbeek** [comment]
Agree. How do we make the community aware of these type of best practices?

**Comment by schuemie** [comment]
The [SqlRender vignette](http://ohdsi.github.io/SqlRender/articles/UsingSqlRender.html#translation-to-other-sql-dialects) might be a good place to capture best practices related to writing SQL that translates properly to the different dialects?

**Comment by PRijnbeek** [comment]
Yes absolutely.
